kubernetes/resource_kubernetes_pod_security_policy.go

```diff
@@ -0,0 +1,509 @@
+package kubernetes
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	api "k8s.io/api/extensions/v1beta1"
```

Comment:
Still using `extensions` here, this needs to be updated to be 

`policy "k8s.io/api/policy/v1beta1"` to match your structured_pod_security_policy.go file

Reply:
Fixed